hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -1245,6 +1245,29 @@ case class LocalizeEntries(child: MatrixIR, entriesFieldName: String) extends Ta
   def execute(hc: HailContext): TableValue = {
     val prev = child.execute(hc)
 
-    TableValue(typ, prev.globals, prev.rvd.updateType(prev.rvd.typ.copy(rowType = newRowType)))
+    TableValue(typ, prev.globals, prev.rvd.cast(newRowType))
+  }
+}
+
+case class TableRename(child: TableIR, rowMap: Map[String, String], globalMap: Map[String, String]) extends TableIR {
```

Comment:
what about `Copy` and `Children` (and others?)

Reply:
those are only for valueIRs -- see the file here for the checklist:

```~/hail/hail/src/main/scala/is/hail/expr/ir/add-ir-checklist.txt```

I did add partitionCounts and a test to TableIRSuite after rereading that.